### DISCUSSION TITLE
Wrong directions?

### DISCUSSION THREAD

**Discussion by DougHall** [Homebrew/discussions#5125] [Everyday usage]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.2.7
ORIGIN: https://github.com/Homebrew/brew
HEAD: 6170b6297c61ef0b68814c708eeff2199d8b0d11
Last commit: 7 days ago
Core tap JSON: 12 Feb 15:41 UTC
Core cask tap JSON: 12 Feb 15:41 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 3.1.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 14.0.0 build 1400
Git: 2.43.1 => /opt/homebrew/bin/git
Curl: 8.4.0 => /usr/bin/curl
macOS: 12.7.3-arm64
CLT: 14.2.0.0.1.1668646533
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

I go to remove an unused openssl@1.1, and here is my output. Pay close attention to the directions it gives me at the very end. (Remove /opt/homebrew/etc/openssl@3 ?????) Surely, this is not right.

[~]$ brew uses --installed openssl@1.1

[~]$ brew remove openssl@1.1
Uninstalling /opt/homebrew/Cellar/openssl@1.1/1.1.1w... (8,101 files, 18MB)

Warning: The following openssl@1.1 configuration files have not been removed!
If desired, remove them manually with `rm -rf`:
  /opt/homebrew/etc/openssl@1.1
  /opt/homebrew/etc/openssl@1.1/cert.pem
  /opt/homebrew/etc/openssl@1.1/ct_log_list.cnf
  /opt/homebrew/etc/openssl@1.1/ct_log_list.cnf.dist
  /opt/homebrew/etc/openssl@1.1/misc
  /opt/homebrew/etc/openssl@1.1/misc/CA.pl
  /opt/homebrew/etc/openssl@1.1/misc/tsget
  /opt/homebrew/etc/openssl@1.1/misc/tsget.default
  /opt/homebrew/etc/openssl@1.1/misc/tsget.pl
  /opt/homebrew/etc/openssl@1.1/misc/tsget.pl.default
  /opt/homebrew/etc/openssl@1.1/openssl.cnf
  /opt/homebrew/etc/openssl@1.1/openssl.cnf.dist

Warning: The following may be openssl@1.1 configuration files and have not been removed!
If desired, remove them manually with `rm -rf`:
  /opt/homebrew/etc/openssl@3

**Comment by roger-scholz** [comment]
Hi DougHall,
Thanks for sharing.
I got the same warning message today ... see below.
This is still irritating, at least to me.
Appreciate any feedback here.
Best, Roger
PS: Did you delete all the stuff or the whole ".../etc/openssl@1.1/" directory?

```
Warning: The following may be openssl@1.1 configuration files and have not been removed!
If desired, remove them manually with `rm -rf`:
  /usr/local/etc/openssl
  /usr/local/etc/openssl@3
```

**Comment by DougHall** [comment]
It’s been a while since I fixed this, but I essentially uninstalled ***@***.*** and installed ***@***.*** I might have had to uninstall everything that was using the @1 version before uninstalling ***@***.*** At that point, yes, you can uninstall the whole ***@***.******@***.***> directory. Then install ***@***.***, and reinstall everything that uses openssl.

This might come in handy:

  *   brew uses ***@***.******@***.***> --installed

From: Roger Scholz ***@***.***>
Date: Thursday, June 20, 2024 at 4:01 AM
To: Homebrew/discussions ***@***.***>
Cc: Doug Hall ***@***.***>, Author ***@***.***>
Subject: Re: [Homebrew/discussions] Wrong directions? (Discussion #5125)
You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>

Hi DougHall,
Thanks for sharing.
I got the same warning message today ... see below.
This is still irritating, at least to me.
Appreciate any feedback here.
Best, Roger
PS: Did you delete all the stuff or the whole ***@***.***/" directory?

Warning: The following may be ***@***.*** configuration files and have not been removed!

If desired, remove them manually with `rm -rf`:

  /usr/local/etc/openssl

  ***@***.***
